src/core/lib/gprpp/global_config_env.h

```diff
@@ -121,7 +121,7 @@ class GlobalConfigEnvString : public GlobalConfigEnv {
   static char g_env_str_##name[] = #name;                                  \
   static ::grpc_core::GlobalConfigEnvString g_env_##name(g_env_str_##name, \
                                                          default_value);   \
-  ::grpc_core::UniquePtr<char> gpr_global_config_get_##name() {            \
+  grpc_core::UniquePtr<char> gpr_global_config_get_##name() {              \
```

Comment:
This is in the macro so it might need to keep `::`

Reply:
Fixed. I've double checked this file and the other ignored files for the purposes of namespace qualification checks, and they are all unchanged from master.